packages/core/LICENSE

```diff
@@ -0,0 +1,21 @@
+Begin license text.
```

Comment:
Given that all packages share the same licence, just keeping the LICENSE file in the top-level directory should be sufficient, I think.

Reply:
Yep - done.